src/routes/components/components.json

```diff
@@ -1161,13 +1155,6 @@
 		"title": "storez",
 		"repository": "https://github.com/plrenaudin/svelte-storez"
 	},
-	{
-		"category": "Stores",
-		"description": "Template for client side svelte store",
-		"tags": ["components and libraries", "stores and state"],
-		"title": "SvelteStore",
-		"repository": "https://github.com/gitbreaker222/SvelteStore"
-	},
```

Comment:
Why removing this packages ?

Reply:
I couldn't find the package on npm - please let me know if you can find it!

Reply:
Hmmm... after further analyzing/reading, it looks like it should be in `templates.json` more than in `components.json` :thinking:

Reply:
I've added SvelteStore to `templates.json` :)